massa-final-state/src/final_state.rs

```diff
@@ -110,10 +104,6 @@ impl FinalState {
         }
         // 5. executed operations hash
         hash_concat.extend(self.executed_ops.hash.to_bytes());
-        debug!(
```

Comment:
You left the cycle history hash debug logs right above this, was it intended?

Reply:
Good catch. Thank you